code/workspaces/auth-service/src/controllers/projectController.js

```diff
@@ -4,6 +4,13 @@ import validator from './validationMiddleware';
 import userRolesRepository from '../dataaccess/userRolesRepository';
 import { PROJECT_ROLES } from '../models/userRoles.model';
 
+async function isMember(req, res) {
+  const { params: { projectKey }, user: { sub } } = req;
+  const exists = await userRolesRepository.userIsMember(sub, projectKey);
+
+  res.status(201).send(exists);
```

Comment:
This should be a 200 as you aren't creating anything and it has body.

Reply:
Will fix.